### ISSUE TITLE
[FactoryLocator] An error has occurred! Game version 0.10.28.20779

### ISSUE THREAD

**Issue by chenglong6622** [starfi5h/DSP_Mod_Support#9]
```ini
An error has occurred! Game version 0.10.28.20779
6 Mods used: [DSP Mod Save1.1.4] [LDBTool3.0.1] [DSP Common API1.5.7.0] [ErrorAnalyzer1.0.0] [FactoryLocator1.2.0] [Super Night Light1.0.6] 
NullReferenceException: Object reference not set to an instance of an object
FactoryLocator.Plugin.Update ();(0)
```

**Comment by starfi5h** [comment]
After CommonAPI is updated, I'll try to fix FactoryLocator

**Comment by starfi5h** [comment]
Fix in FactoryLocator v1.2.1
